.github/ISSUE_TEMPLATE/content-proposal.md

```diff
@@ -17,8 +17,36 @@ A short summary of the audience. Include:
 - Expertise in the particular topic to be discussed
 
 ## Level of content
-Level of content as it pertains to the topic proposed
-- [x] 101: "Awareness / Exploration"
-- [ ] 201: "Prototyping / Deploying"
-- [ ] 301: "Scaling
-- [ ] Content Series: To include multiple levels of content (Note: Series pieces should either be able to be consumed independently, or very explicit in the title, and with links in the body of the content that they are part of a series)
+Level of content as it pertains to the topic proposed. Delete those not applicable.
+- 101: "Awareness / Exploration"
+- 201: "Prototyping / Deploying"
+- 301: "Scaling
+- Content Series: To include multiple levels of content (Note: Series pieces should either be able to be consumed independently, or very explicit in the title, and with links in the body of the content that they are part of a series)
+
+## Contributor Checklist
+1. [x] I have added a card to the [GitHub Content Backlog](https://github.com/vmware-tanzu/tanzu-dev-portal/projects/7) and/or moved it to the proper column. See [Content Prewriting and Ideation](https://github.com/vmware-tanzu/tanzu-dev-portal/wiki/Contribution,-Prewriting-Ideation-(GitHub-and-Google-Docs-Method))
+2. [ ] **If contributing via Google Docs**: I have added a link to my document to this issue, and made is visible with comment rights to everyone.
```

Comment:
```suggestion
2. [ ] **If contributing via Google Docs**: I have added a link to my document to this issue, and set the document's Share settings to "Anyone on the internet with this link can comment".
```

Reply:
![Screen_Shot_2021-08-04_at_3_19_22_PM](https://user-images.githubusercontent.com/13995/128256780-a6747cf7-e2d8-4c79-ab8d-ebb861e2d8e2.png)

Consider referencing this or similar screenshot.